config/crd/bases/galaxy_v1beta1_galaxyrestore_crd.yaml

```diff
@@ -62,8 +62,8 @@ spec:
                   type: string
                 no_log:
                   default: true
-                  description: Configure no_log for no_log tasks
-                  type: boolean
+                  description: Configure no_log (hides sensitive information in operator/task logs)
+                  type: boolean 
```

Comment:
extra whitespace to be removed. not a big deal, we can catch with a linter later if wanted